app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,317 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MovieDetailsActivity"
+    android:background="@color/bg_dark">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="20dp">
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/left_guidline"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:orientation="vertical"
+            app:layout_constraintGuide_begin="16dp">
+
+        </androidx.constraintlayout.widget.Guideline>
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/right_guidline"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:orientation="vertical"
+            app:layout_constraintGuide_end="16dp">
+
+        </androidx.constraintlayout.widget.Guideline>
+
+        <ImageView
+            android:id="@+id/IV_bg_image"
+            android:layout_width="match_parent"
+            android:layout_height="298dp"
+            android:scaleType="centerCrop"
+            android:src="@drawable/bg_image"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:background="@drawable/image_mask"
+            app:layout_constraintBottom_toBottomOf="@id/IV_bg_image"
+            app:layout_constraintEnd_toEndOf="@id/IV_bg_image"
+            app:layout_constraintStart_toStartOf="@id/IV_bg_image"
+            app:layout_constraintTop_toTopOf="@id/IV_bg_image" />
+
+        <ImageView
+            android:id="@+id/IV_back_with_arrow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="56dp"
+            android:src="@drawable/btn_backs_arrow"
+            android:padding="0dp"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toTopOf="@id/IV_bg_image"
+            app:tint="@color/gray" />
+        <TextView
+            android:id="@+id/TV_back_with_arrow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="56dp"
+            android:text="@string/back_with_arrow"
+            android:textSize="14sp"
+            android:textColor="@color/gray"
+            app:layout_constraintStart_toEndOf="@id/IV_back_with_arrow"
+            app:layout_constraintTop_toTopOf="@id/IV_bg_image" />
+
+        <TextView
+            android:id="@+id/TV_age_limit"
```

Comment:
think about bigger text in TV_movie_name(it will overdraw some views)
looks like logically: TV_age_limit should be above TV_movie_name
do it